README

```diff
@@ -156,21 +156,47 @@ REQUIREMENTS:
         dracut (optional)
         PolicyKit (optional)
 
-        When building from git, the following tools are needed:
+        Two build systems are supported: meson + ninja-build and autools + make.
+
+        The following tools are needed with both systems:
 
         pkg-config
-        docbook-xsl
-        xsltproc
+        gperf >= 3.1
+        docbook-xsl (optional, required for documentation)
+        xsltproc    (optional, required for documentation)
+        python-lxml (optional, required to build the indices)
+
+        When building with meson, python and ninja-build are required.
```

Comment:
The project is called ninja-build, even though the binary is called just ninja. Here I'm referring to the project.

meson on EPEL is using python2, so I think there's no reason to specify python version. You get some version pulled in through deps, it's up to your distro and we don't really care.